chains/build_spec.js

```diff
@@ -97,6 +97,7 @@ async function setAuthorities(chainSpec, chainConfig, opts) {
 async function setStarport(chainSpec, chainConfig, opts) {
   let chainDeployment = await fetchChainDeployment(chainConfig.eth_network);
   let starportAddress = must(chainDeployment.Contracts, 'Starport');
+  // TODO: Set other properties
```

Comment:
TODO: We should probably remove properties.

Reply:
I'm pretty sure the `properties` key is buried somewhere in substrate but we can remove all of the `properties` that we looked for.